arangod/Aql/OptimizerRules.cpp

```diff
@@ -3932,12 +3931,14 @@ auto insertGatherNode(
 
       gatherNode = plan.createNode<GatherNode>(&plan, plan.nextId(), sortMode,
                                                parallelism);
-    } break;
+      break;
```

Comment:
Why did you this change?

Reply:
I changed 3 occurences of `break` from `{ ... } break;` to { ... break; }` that were all close to places where I removed unreachable code. I find the `{ ... break; }` version more readable than the other variant.
Feel free to push a change into this PR if you don't like that style.